osquery/tables/applications/windows/carbon_black.cpp

```diff
@@ -15,6 +15,7 @@
 
 #include <osquery/core.h>
 #include <osquery/filesystem.h>
+#include <osquery/filesystem/fileops.h>
```

Comment:
Is this file in the global-search path `./include` or local? If it's within `./osquery` then place it below the global-search includes and use `""` for the path.